src/Pages/CustomDatePhoto.js

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { StyledInput } from "../Utils/Styled Components/StyledInput";
+import { StyledTitle, StyledDescription } from "../Utils/Styled Components/StyledText";
+import { StyledImgContainer, StyledImg, StyledImgText } from "../Utils/Styled Components/StyledImg";
+import { StyledIFrame } from "../Utils/Styled Components/StyledIFrame";
+
+export default function CustomDatePhoto() {
+
+    let initDate = {
+        target: {
+            value: "2022-01-19"
+        }
+    }
```

Comment:
Why don't you store value directly? 
```suggestion
    const initDate = "2022-01-19";
```